commands/display.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+
+	name: "display",
+	description: "Displays variable without editing code! (Used for debugging)",
+	args: true,
+	usage: `var`,
+    allowPM: true,
+    
+	execute(message, args) {
+
+		eval(`message.channel.send(${args.join(" ")})`);
```

Comment:
to jest w huj niebezpieczne!!!
nie możemy mieć komendy która odpala js w discordzie

Reply:
TBH, nie jesteś w stanie za wiele w tym zrobić. Każdy średnik psuje tą komendę i wywala błąd, więc jedyne co działa to pojedyncze zmienne.

Reply:
challenge accepted

Reply:
yep its not secure